### PR TITLE
pythonPackages.pyfribidi: fix build

### PR DESCRIPTION
**Pull request by maxxk**
Add missing dependency: six.
Pyfribidi 0.12.0 supports Python 3.

###### Motivation for this change

ZHF: #80379

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build pythonPackages.pyfribidi python38Packages.pyfribidi python37Packages.pyfribidi python36Packages.pyfribidi python35Packages.pyfribidi

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build pythonPackages.pyfribidi

**Review by jonringer — APPROVED** [review]
LGTM for x64-linux

```
[4 built, 1 copied (0.7 MiB), 0.7 MiB DL]
https://github.com/NixOS/nixpkgs/pull/80838
3 package built:
python27Packages.pyfribidi python37Packages.pyfribidi python38Packages.pyfribidi
```
might be nice to address darwin issues (or disable, looks like upstream issues)

File: pkgs/development/python-modules/pyfribidi/default.nix [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-, isPy3k
 , isPyPy
+, six
 }:
 
 buildPythonPackage rec {
   version = "0.12.0";
   pname = "pyfribidi";
-  disabled = isPy3k || isPyPy;
+  disabled = isPyPy;
```
Comment (jonringer) [inline pkgs/development/python-modules/pyfribidi/default.nix]:
```suggestion
  disabled = isPyPy || stdenv.cc.isClang;
```
quick and easy way to disable failing clang builds

Comment (maxxk) [reply pkgs/development/python-modules/pyfribidi/default.nix]: Thank you! In this case build log looks like there is a single, easy to fix error. Maybe small patch will be enough.

File: pkgs/development/python-modules/pyfribidi/pyfribidi-clang.patch [hunk 2]
```diff
@@ -0,0 +1,17 @@
+diff --git a/pyfribidi.c b/pyfribidi.c
+index 9a0120d..238134a 100644
+--- a/pyfribidi.c
```
Comment (JohnAZoidberg) [inline pkgs/development/python-modules/pyfribidi/pyfribidi-clang.patch]: Nice, could you please try to upstream this?

**Pull request merged**